jsk_unitree_robot/cross/build_ros1_dependencies.sh

```diff
@@ -45,5 +45,6 @@ docker run -it --rm \
     pip install -U --user pip && \
     export PYTHONPATH=\"/opt/jsk/System/ros1_dependencies/lib/python2.7/site-packages\" && \
     export PKG_CONFIG_PATH=\"/opt/jsk/${INSTALL_ROOT}/ros1_dependencies/lib/pkgconfig\" && \
+    pip install spidev==3.5 pixel-ring==0.1.0 && \
```

Comment:
We try to separate ‘data’ and ‘processing’, so we will write which modules to install in [go_requirements.txt](https://github.com/jsk-ros-pkg/jsk_robot/pull/1794/commits/b639039ba09c54bdd71361b3fbf05cd08749e91a) and try to build_ros_dependencies.sh without depending on installed modules or even unitree robot. This way, we can integrate this file with Pepper cross-environment near future.